tests/sets/tsets.nim

```diff
@@ -173,7 +173,7 @@ type
     warnProveInit, warnProveField, warnProveIndex, warnGcUnsafe, warnGcUnsafe2,
     warnUninit, warnGcMem, warnDestructor, warnLockLevel, warnResultShadowed,
     warnUser,
-    hintSuccess, hintSuccessX,
+    hintSuccess, hintSuccessX, hintBuildMode,
```

Comment:
remove this diff, unrelated (refs https://github.com/nim-lang/Nim/pull/17876 which will avoid this in future)

Reply:
done.